lib/type.ts

```diff
@@ -521,17 +521,37 @@ function arrayEquals(a : unknown[]|null, b : unknown[]|null) : boolean {
     return true;
 }
 
-const DEFAULT_ENTITY_SUB_TYPE : Record<string, string> = {
-    'tt:picture': 'tt:url'
+const DEFAULT_ENTITY_SUB_TYPE : Record<string, string[]> = {
+    'tt:picture': ['tt:url']
 };
 
-function entitySubType(type : string, assignableTo : string, entitySubTypeMap : EntitySubTypeMap) : boolean {
+function _getAncestors(type : string, entitySubTypeMap : EntitySubTypeMap) : string[] {
```

Comment:
This doesn't seem very efficient, should we cache it somehow?

Reply:
fixed in c33993f